.github/workflows/release-please.yml

```
@@ -61,3 +61,28 @@ jobs:
+  publish-wren-langchain:
+    needs: release-please
+    if: needs.release-please.outputs['wren-langchain--release_created'] == 'true'
+    uses: ./.github/workflows/publish-wren-langchain.yml
+    with:
+      version: ${{ needs.release-please.outputs['wren-langchain--version'] }}
+      tag_name: ${{ needs.release-please.outputs['wren-langchain--tag_name'] }}
+    permissions:
+      contents: read
+      id-token: write
+
+  publish-wren-langchain:
```

actionlint (1.7.12)

[error] key "publish-wren-langchain" is duplicated in "jobs" section. previously defined at line:68,col:3. note that this key is case insensitive

(syntax-check)